### DISCUSSION TITLE
Question about the Planet Orbit

### DISCUSSION THREAD

**Discussion by coctree52** [Stellarium/stellarium#1649] [Q&A] [answered]
By chance I found this question and I cannot figure out why.

![stellarium-000](https://user-images.githubusercontent.com/83047344/117544331-88dea980-b053-11eb-81f9-a1f7050aedd6.jpg)
![stellarium-001](https://user-images.githubusercontent.com/83047344/117544338-8da35d80-b053-11eb-88a6-0651e39ed504.jpg)
![stellarium-002](https://user-images.githubusercontent.com/83047344/117544342-909e4e00-b053-11eb-908f-1bf9d7547b38.jpg)

I was standing on Ganymede, with both "atmosphere" and "ground" off. When I selected Jupiter and let the orbit show, it came to me like these. 

I have no idea why the revolving orbit can twist in such a weird manner.  Maybe I misunderstood these kind of "orbits," but what on earth are they symbolizing?

**Comment by alex-w** [comment, marked as answer]
This is a bug

**Reply by coctree52** [reply]
Thank you for your timely reply and fortunately the question won't bother with my sleep this night...

**Comment by coctree52** [comment]
I am sorry but for one year and a half this problem is still unsolved

**Comment by gzotti** [comment]
Technically the answer was accepted, case closed. 

Without checking the code: Orbit is plotted with a fixed number of points, connected with straight lines. The one long straight line is a perspective effect.

**Reply by coctree52** [reply]
I rarely browse this website and I remember this "discussion" used to be an "issue" and was labeled as "bug" even though I accepted the answer. Now I came back and found that the label had disappeared, but the orbits mentioned above still look weird. Can you help resolving this problem in the future?

**Reply by gzotti** [reply]
Given that the case was reopened 1 day ago and is still not a bug report, we also see it, but only rarely,. and may be able to eventually improve it, although it is not my top priority.
